src/features/landing/components/Header/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { AppBar, Toolbar } from '@mui/material';
+
+import VilnaMainIcon from '../Icons/VilnaMainIcon/VilnaMainIcon';
+
+import Buttons from './Buttons/Buttons';
+import NavLink from './NavLink/NavLink';
+
+const links = [
+  { id: 1, value: 'Переваги' },
+  { id: 2, value: 'Для кого' },
+  { id: 3, value: 'Інтеграція' },
+  { id: 4, value: 'Контакти' },
+];
+
+function Header() {
+  return (
+    <AppBar
+      position="static"
+      sx={{ backgroundColor: 'white', boxShadow: 'none' }}
```

Comment:
don't use colors like this, we need only one type of how we can write it, for example #ffffff, and we need to create one place for all colors that we used across all application